VelocidiSDK.podspec

```diff
@@ -0,0 +1,138 @@
+#
+#  Be sure to run `pod spec lint VelocidiSDK.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
```

Comment:
This file seems complete. Are the explanatory comments still necessary?

Reply:
This comments came by default. Also thought about removing them, but decided in favor of leaving them because it might be useful in these early stages. We can remove them later.